kickstarter/android-oss
# WEB-748 :  [Android] Initial Page Viewed event not firing

# 📲 What
After our Segment hot fix we stopped firing the initial Page Viewed (Discovery) event when the app is opened.

# 🛠 How
Add delay to the event 


| Before 🐛  |
<img width="1039" alt="image" src="https://user-images.githubusercontent.com/1075310/193574395-c2968b0b-1f71-4924-9a73-2a0a923113cc.png">

| After 🦋 |
<img width="1055" alt="image" src="https://user-images.githubusercontent.com/1075310/193574050-54f5f5cd-1e14-4f5b-b943-7b66fa8c54d9.png">

# 📋 QA

Steps to reproduce

1. Open Segment debugger
2. Open the app
3. Check there's an `Application Opened` event
4. Check `Page Viewed` for the Discovery page

# Story 📖

[[Name of Trello Story](Trello link)](https://kickstarter.atlassian.net/browse/WEB-748)
